.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,44 @@
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          projectBaseDir: .
+          args: >
+            -Dsonar.projectKey=your-project-key
+            -Dsonar.sources=.
+            -Dsonar.tests=backend/src/__tests__
+            -Dsonar.javascript.lcov.reportPaths=backend/coverage/lcov.info
+            -Dsonar.test.inclusions=backend/src/__tests__/**/*.test.ts
+            -Dsonar.exclusions=**/node_modules/**,**/coverage/**,**/dist/**,**/build/**
+            -Dsonar.coverage.exclusions=**/node_modules/**,**/coverage/**,**/dist/**,**/build/**,**/__tests__/**,**/interfaces/**,**/config/**,**/enums/**,**/errors/**,**/helpers/**
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)